zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java

```diff
@@ -322,7 +319,8 @@ public boolean runParagraph(String noteId,
 
     try {
       notebook.saveNote(note, context.getAutheInfo());
-      boolean result = note.run(p.getId(), blocking);
+//      boolean result = note.run(p.getId(), blocking);
```

Comment:
Unwanted code can be deleted directly.

Reply:
@srehmann , Please delete `//      boolean result = note.run(p.getId(), blocking);`